components/learn/modules/ROOT/pages/public-staging.adoc

```diff
@@ -96,8 +103,6 @@ module.exports = {
 };
 ----
 
-NOTE: The `HDWalletProvider` can be configured to generate more than a single account from the mnemonic. Check out its https://github.com/trufflesuite/truffle/tree/master/packages/hdwallet-provider[documentation] for more info.
```

Comment:
I'm not against removing the notice about multiple accounts, but we should add the link to their documentation on the other mention of the hdprovider then.

Reply:
Updated note to be generic